### PR TITLE
core(cmd): adding corrections to cmd

### PR DESCRIPTION
**Pull request by dwertent**
## Overview
* Removed deprecated commands
* Fixed some command descriptions

## Update output
Replacing the text status with icons:
image

Highest-stake workloads:
image

cc @craigbox

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -74,22 +72,22 @@ func getRootCmd(ks meta.IKubescape) *cobra.Command {
...
 	rootCmd.AddCommand(download.GetDownloadCmd(ks))
-	rootCmd.AddCommand(delete.GetDeleteCmd(ks))
 	rootCmd.AddCommand(list.GetListCmd(ks))
-	rootCmd.AddCommand(submit.GetSubmitCmd(ks))
 	rootCmd.AddCommand(completion.GetCompletionCmd())
 	rootCmd.AddCommand(version.GetVersionCmd())
 	rootCmd.AddCommand(config.GetConfigCmd(ks))
 	rootCmd.AddCommand(update.GetUpdateCmd())
 	rootCmd.AddCommand(fix.GetFixCmd(ks))
+	// rootCmd.AddCommand(submit.GetSubmitCmd(ks))// the submit command should be supported after we add a third-party authentication in Kubescape
```
Comment (dwertent) [inline cmd/root.go:89]: @amirmalka I do not see any reason for these commands

Comment (amirmalka) [reply cmd/root.go:89]: We still had Submit results no?

[the remark this answers is not in the document]
Comment (dwertent) [reply cmd/root.go:89]: We can work with something like this :)

**Review by Daniel-GrunbergerCA — DISMISSED** [review]

**Comment by Daniel-GrunbergerCA** [comment]
Note that you need to update the tests accordingly

**Review by amirmalka — APPROVED** [review]

**Review by matthyx — APPROVED** [review]

**Pull request merged**